File: c/src/common/bostonclamchowder.c
#include "bostonclamchowder.h"

soup_t * bostonclamchowder_new()
{
	soup_t * soup = soup_new();
	soup->name = "QuahogChowder";
	arraylist_string_add( soup->ingredients,"1 Pound Fresh Quahogs");
	arraylist_string_add( soup->ingredients,"1 cup corn");    
	arraylist_string_add( soup->ingredients,"1/2 cup heavy cream");
	arraylist_string_add( soup->ingredients,"1/4 cup butter");    
	arraylist_string_add( soup->ingredients,"1/4 cup potato chips");
	return soup;
}

File: php/src/common/BostonClamChowder.php
<?

require_once 'ClamChowder.php';


class BostonClamChowder extends ClamChowder
{

	function __construct()
	{
		$this->soupName = "QuahogChowder";
		$this->soupIngredients = [
        "1 Pound Fresh Quahogs",
        "1 cup corn",
        "1/2 cup heavy cream",
        "1/4 cup butter",
        "1/4 cup potato chips",
		];
	}

}
